Sources/ParseSwift/Object Protocols/ParseInstallation.swift

```diff
@@ -217,6 +217,7 @@ extension ParseInstallation {
             return
         }
 
+        #if !os(Linux)
```

Comment:
Same here; and the next part isn't in this PR, but are you sure the desired behavior is for the Mac Catalyst bundle ID prefix to be dropped? Unless the ObjC SDK is doing the same thing, my opinion is that if Apple thinks the Catalyst app is different enough to warrant a new bundle ID, the SDK should probably respect that.

Reply:
For the part, same as above.

In reference to Catalyst, I did the exact same fix the obj-c SDK does here:

https://github.com/parse-community/Parse-SDK-iOS-OSX/blob/d0119bd7602d4b880b0038b2ac2b9246044b8564/Parse/Parse/PFInstallation.m#L302-L316